EssentialCSharp.Web.Tests/RouteConfigurationServiceTests.cs

```diff
@@ -0,0 +1,65 @@
+using EssentialCSharp.Web.Services;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace EssentialCSharp.Web.Tests;
+
+public class RouteConfigurationServiceTests : IClassFixture<WebApplicationFactory<Program>>
+{
+    private readonly WebApplicationFactory<Program> _Factory;
+    private readonly IRouteConfigurationService _RouteConfigurationService;
+
+    public RouteConfigurationServiceTests(WebApplicationFactory<Program> factory)
+    {
+        _Factory = factory;
+
+        // Get the service from the DI container to test with real routes
+        var scope = _Factory.Services.CreateScope();
+        _RouteConfigurationService = scope.ServiceProvider.GetRequiredService<IRouteConfigurationService>();
+    }
+
+    [Fact]
+    public void GetStaticRoutes_ShouldReturnExpectedRoutes()
+    {
+        // Act
+        var routes = _RouteConfigurationService.GetStaticRoutes().ToList();
+
+        // Assert
+        Assert.NotEmpty(routes);
+
+        // Check for expected routes from the HomeController
+        Assert.Contains("home", routes);
+        Assert.Contains("about", routes);
+        Assert.Contains("guidelines", routes);
+        Assert.Contains("announcements", routes);
+        Assert.Contains("termsofservice", routes);
+    }
+
+    [Fact]
+    public void GetStaticRoutes_ShouldIncludeAllHomeControllerRoutes()
```

Comment:
This test appears to be a dup of the one above it. Is it testing anything different?

Reply:
Leftover from debugging, removed